### PR TITLE
Dev/editpageUI : UI changes in edit page 

### PR DESCRIPTION
**Pull request by Pendyala-thanvi**
### Functionality:
edit page

### Solution:
new UI changes

### Risk level:
- [ ] high 
- [x] medium
- [ ] low

### How to test:
ng serve
firebase emulators:start

### CODE REVIEW

File: src/app/body/task-details/edit-page/edit-page.component.html [hunk 1]
```diff
@@ -1,165 +1,107 @@
...
-                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
-                            <span aria-hidden="true" class="material-icons text-light"(click)="editTaskDone()">close</span>
-                        </button>
-                    </li>
-                </ul>
-            </nav>
-            <div class="container modal-body" id="editPageTaskContainer">
+<div class="modal fade" id="exampleModalCenter" tabindex="-1" role="dialog" aria-labelledby="ModalCenterTitle" aria-hidden="true">
+    <div class="modal-dialog modal-dialog-centered" role="document">
+        <div class="row modal-content">
```
Comment (k0malSharma) [inline src/app/body/task-details/edit-page/edit-page.component.html:3]: Don't add row with modal content. Let them be in separate div.

File: src/app/body/task-details/edit-page/edit-page.component.html [hunk 2]
```diff
@@ -1,165 +1,107 @@
...
+            <div class="modal-header pt-3">
+                <div class="col-8 col-md-8 pl-2">
+                    <h5 class="modal-title pl-3" id="exampleModalCenterLabel">Edit Task</h5>
+                    <span class="pl-3" style="color: var(--text);">{{ todayDate }}</span>
```
Comment (k0malSharma) [inline src/app/body/task-details/edit-page/edit-page.component.html:7]: Don't  apply the color style here

File: src/app/body/task-details/log-work/log-work.component.html [hunk 3]
```diff
@@ -1,8 +1,7 @@
-<div class="container-fluid">
 <div class="modal fade" id="Modal" tabindex="-1" role="dialog" aria-labelledby="ModalCenterTitle" aria-hidden="true">
     <div class="modal-dialog modal-dialog-centered" role="document">
-        <div class="modal-content">
-            <div class="modal-header row pt-3">
+        <div class="row modal-content">
```
Comment (k0malSharma) [inline src/app/body/task-details/log-work/log-work.component.html:3]: Do not apply row with modal content. Use different div for both

**Review by k0malSharma — CHANGES_REQUESTED** [review]

**Review by viv9k — APPROVED** [review]
Approved

**Pull request merged**